Makefile

```diff
@@ -7,7 +7,7 @@ parse: _check_python
 lint: node_modules
 	@python ./tools/specification_parser/lint_json_output.py specification.json
 	./node_modules/.bin/markdownlint --ignore node_modules/ --ignore tools/ **/*.md
-	./node_modules/.bin/markdown-link-check -c .markdown-link-check-config.json README.md specification/*.md
+	./node_modules/.bin/markdown-link-check -c .markdown-link-check-config.json README.md specification/*.md specification/**/*.md
```

Comment:
Also recursively check spec subfolders.

Reply:
sadly this won't prevent this issue currently - but hopefully in the near future see https://github.com/tcort/markdown-link-check/issues/225

Reply:
Ya, no problem. This will at least validate the non-anchor parts of links in docs in subfolders, so it's a step in the right direction.